examples/landspill/inclined_plane/setrun.py

```diff
@@ -0,0 +1,417 @@
+    amrdata.uprint = False      # update/upbnd reporting
+
+    # More AMR parameters can be set -- see the defaults in pyclaw/data.py
+
+    # == setregions.data values ==
+    regions = rundata.regiondata.regions
+    # to specify regions of refinement append lines of the form
+    #  [minlevel,maxlevel,t1,t2,x1,x2,y1,y2]
+
+    # == setgauges.data values ==
+    # for gauges append lines of the form  [gaugeno, x, y, t1, t2]
+    # rundata.gaugedata.gauges.append([])
+
+    return rundata
+    # end of function setrun
+    # ----------------------
+
+
+#-------------------
+def setgeo(rundata):
+#-------------------
+    """
+    Set GeoClaw specific runtime parameters.
+    For documentation see ....
+    """
+
+    try:
+        geo_data = rundata.geo_data
+    except:
+        print("*** Error, this rundata has no geo_data attribute")
+        raise AttributeError("Missing geo_data attribute")
+
+
+    # == Physics ==
+    geo_data.gravity = 9.81
+    geo_data.coordinate_system = 1
+    geo_data.earth_radius = 6367.5e3
+    geo_data.sea_level = -1000.
+
+    # == Fluid ==
+    geo_data.rho = 970. # kg / m^3
+    geo_data.nu = 1.13e-3 # m^2 / sec
+
+    # == Forcing Options
+    geo_data.coriolis_forcing = False
+
+    # == Algorithm and Initial Conditions ==
+    geo_data.sea_level = 0.0
+    geo_data.dry_tolerance = 1e-5
+    geo_data.friction_forcing = False
+    geo_data.manning_coefficient = 0.035
+    geo_data.friction_depth = 1.e6
+
+    # Refinement data
+    refinement_data = rundata.refinement_data
+    refinement_data.wave_tolerance = 1.e-5
+    refinement_data.deep_depth = 1e2
+    refinement_data.max_level_deep = 3
+    refinement_data.variable_dt_refinement_ratios = True
+
+    # == settopo.data values ==
+    topo_data = rundata.topo_data
+    # for topography, append lines of the form
+    #    [topotype, minlevel, maxlevel, t1, t2, fname]
+    topo_data.topofiles.append([3, 1, 10, 0., 1.e10, 'inclined_plane_2.5.txt'])
+
+    # == setdtopo.data values ==
+    dtopo_data = rundata.dtopo_data
+    # for moving topography, append lines of the form :   (<= 1 allowed for now!)
+    #   [topotype, minlevel,maxlevel,fname]
+
+    # == setqinit.data values ==
+    rundata.qinit_data.qinit_type = 0
+    rundata.qinit_data.qinitfiles = []
+    # for qinit perturbations, append lines of the form: (<= 1 allowed for now!)
+    #   [minlev, maxlev, fname]
+
+    # == setfixedgrids.data values ==
+    fixedgrids = rundata.fixed_grid_data
+    # for fixed grids append lines of the form
+    # [t1,t2,noutput,x1,x2,y1,y2,xpoints,ypoints,\
+    #  ioutarrivaltimes,ioutsurfacemax]
+
+    # Point sources
+    from clawpack.geoclaw.data import PointSourceData
+    rundata.add_data(PointSourceData(), 'pointsource_data')
+    ptsources_data = rundata.pointsource_data
+    ptsources_data.n_point_sources = 1
+    ptsources_data.point_sources.append(
+        [[0., 0.], 1, [3600.], [1.48e-6]])
+
+    from clawpack.geoclaw.data import DarcyWeisbachData
```

Comment:
Similar to a comment made above, may be better to just include this friction as an alternative friction source term rather than a completely separate object.